### ISSUE TITLE
Upgrading the app to current day

### ISSUE THREAD

**Issue by TeemuKoivisto** [arjanfrans/realtime-multiplayer-in-html5#18]
Hey Arjan, nice work 👍! I just forked your repo and refactored it to use pnpm monorepo and ES modules https://github.com/TeemuKoivisto/realtime-multiplayer-in-html5 However, I can't seem to figure out how to fix the events between the client and the server to actually launch the game. If you or anyone has any idea how to fix it, would be a lot appreciated if you could take a look. It would be cool to be able to run this game again and I'll set up a working site and backend if we accomplish that 😃

**Comment by arjanfrans** [comment]
Hey, I haven't checked this code for years, so I am not sure what the problem could be.

I'll try to run your code some time this week and see what is happening.

**Comment by TeemuKoivisto** [comment]
Hah yeah, 7 years huh. I did rewrite the original version in my repo almost fully. However, the further I got into the process I realized the way it passed inputs to clients instead of just vectors was little awkward. I implemented a similar interpolation with deltatime though in my own project so I suppose it was worthwhile. No need to spend too much time on this though, sometimes perhaps it's best to just archive a project once it's too much hassle to maintain 😁.
